bot/cogs/watchchannels/watchchannel.py

```diff
@@ -287,7 +287,9 @@ async def send_header(self, msg: Message) -> None:
 
         await self.webhook_send(embed=embed, username=msg.author.display_name, avatar_url=msg.author.avatar_url)
 
-    async def list_watched_users(self, ctx: Context, update_cache: bool = True) -> None:
+    async def list_watched_users(
+        self, ctx: Context, oldest_first: bool = False, update_cache: bool = True
+    ) -> None:
```

Comment:
Method signature still fits within 120 chars, is this broken down intentionally? I don't mind it, just wondering.

Additionally, the `oldest_first` param isn't documented in the docstring (it is in the commands), but I think it's self-explanatory and doesn't need to be mentioned.